src/mds/journal.cc

```diff
@@ -1681,6 +1742,7 @@ void ESession::encode(bufferlist &bl, uint64_t features) const
   encode(inos, bl);
   encode(inotablev, bl);
   encode(client_metadata, bl);
+  encode(purge_inos, bl);
```

Comment:
Ah, you need to bump the encoding version above:

`ENCODE_START(6, 5, bl);`

Reply:
updated it. thx